arcgis-ios-sdk-samples/Utility network/Create load report/CreateLoadReportViewController.swift

```diff
@@ -0,0 +1,360 @@
+        SVProgressHUD.show(withStatus: "Loading utility network…")
+        utilityNetwork.load { [weak self] error in
+            SVProgressHUD.dismiss()
+            guard let self = self else { return }
+            guard error == nil else {
+                self.presentAlert(error: error!)
+                return
+            }
+            // Create a default starting location.
+            guard let startingLocation = self.makeStartingLocation() else {
+                self.presentAlert(message: "Fail to create starting location.")
+                return
+            }
+            // Get the base condition and trace configuration from a default tier.
+            guard let utilityTierConfiguration = self.getTraceConfiguration() else {
+                self.presentAlert(message: "Fail to get trace configuration.")
+                return
+            }
+            // Proceed if the utility network loaded without issue.
+            self.utilityNetworkDidLoad(startingLocation: startingLocation, traceConfiguration: utilityTierConfiguration)
+        }
+    }
+    
+    /// Called in response to the utility network load operation completing.
+    /// - Parameters:
+    ///   - startingLocation: The utility element to start the trace from.
+    ///   - traceConfiguration: The utility tier's trace configuration.
+    func utilityNetworkDidLoad(startingLocation: AGSUtilityElement, traceConfiguration: AGSUtilityTraceConfiguration) {
+        // Set the default expression.
+        initialExpression = traceConfiguration.traversability?.barriers as? AGSUtilityTraceConditionalExpression
+        
+        // Create downstream trace parameters with function outputs.
+        let traceParameters = AGSUtilityTraceParameters(traceType: .downstream, startingLocations: [startingLocation])
+        traceParameters.resultTypes.append(.ags_value(with: .functionOutputs))
+        
+        // The service category for counting total customers.
+        if let serviceCategory = utilityNetwork.definition.categories.first(where: { $0.name == "ServicePoint" }),
+           // The load attribute for counting total load.
+           let loadAttribute = utilityNetwork.definition.networkAttributes.first(where: { $0.name == "Service Load" }),
+           // The phase attribute for getting total phase current load.
+           let phasesNetworkAttribute = utilityNetwork.definition.networkAttributes.first(where: { $0.name == "Phases Current" }) {
+            self.phasesNetworkAttribute = phasesNetworkAttribute
+            // Get possible coded phase values from the attributes.
+            if let domain = phasesNetworkAttribute.domain as? AGSCodedValueDomain {
+                excludedPhases = domain.codedValues.sorted { $0.name < $1.name }
+                DispatchQueue.main.async { [weak self] in
+                    guard let self = self else { return }
+                    self.tableView.reloadSections([Section.excluded.rawValue], with: .automatic)
```

Comment:
You could help yourself out here (and elsewhere) with an extension on `UITableView`:

```swift
private extension UITableView {
    func reloadSection(_ section: CreateLoadReportViewController.Section) {
        reloadSections([section.rawValue], with: .automatic)
    }
}

self.tableView.reloadSection(.excluded)
```